app/views/games/new.html.erb

```diff
@@ -0,0 +1 @@
+<h1>New Game</h1>
```

Comment:
There should be a form here for the game and available fields to fill out.

`<%= simple_form_for @game do |f| %>` 
etc